requirements-lint.txt

```
@@ -1,4 +1,5 @@
 argcomplete==2.0.0
+astroid==2.12.12
```

I guess we'll want to address the real pylint issue at some point, but right now this should fix it:

```suggestion
astroid==2.12.11
```

---

That worked. Thanks.